src/css/components/select.css

```diff
@@ -4,7 +4,7 @@
 
 select {
     -moz-appearance: none;
-    background-color: var(--button-background);
+    background-color: #4A4A4F;
```

Comment:
@jhonatasrm can you create a CSS variable because using a hardcoded value will break the light theme

Reply:
Yes... I will look at the settings.css 
🙂

Reply:
I fixed the missing values for the light theme.